src/style.css

```
@@ -1,3 +1,62 @@
 body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  display: inline-block;
+}
```

create class for `img`
```suggestion
img {
  width: 40px;
  height: 40px;
}
```